docs/running-node.md

```diff
@@ -342,3 +342,85 @@ As a general guidance you may consider to use only the peers you own and/or trus
+  - Steps:
+    - Add a node with a binary version as was at the genesis state
+    - Let the nodes catch-up and play all updates/migrations
+  
+  - Pros:
+    - The new node contains all the blocks from blockchain (full history)
+
+  - Cons:
+    - Can take quite a lot of time (depends on blockchain size)
+    - Probably error-prone (if at least one migration has bug catchup fails)
+
+2) Take the data from another node
+- Prerequisites:
+  - Access to a machine running an up-to-date node to copy data from
+
+- Steps:
+  - Stop running an up-to-date node (to have consistent data), and copy the data folder
+  - Put that data to the new node's data folder (can be automated)
+  - Run the new node
+
+- Pros:
+  - No specific node configuration needed (just start the new node)
+
+- Cons:
+  - Needs an access to another machine (may not be possible to everyone)
+  - Node that data is copied from should be stopped (downtime)
+  - Probably error prone due to manual operations 
+
+* References:
+  - https://docs.cosmos.network/master/core/upgrade.html#syncing-a-full-node-to-an-upgraded-blockchain
+3) State sync (Recommended)
+- Prerequisites:
+  - Latest binary version running in existing network
+  - State sync snapshots should be enabled in at least one node in the network
+    - The following parameters should be set in `$HOME/.dcl/config/app.toml`:
+      ```
+      # snapshot-interval specifies the block interval at which local state sync snapshots are
+      # taken (0 to disable). Must be a multiple of pruning-keep-every.
+      snapshot-interval = 1000
+
+      # snapshot-keep-recent specifies the number of recent snapshots to keep and serve (0 to keep all).
+      snapshot-keep-recent = 2
+      ```
+      > **_NOTE:_** Snapshot interval must currently be a multiple of the pruning-keep-every(default 100), to prevent heights from being pruned while taking snapshots. It’s also usually a good idea to keep at least 2 recent snapshots, such that the previous snapshot isn’t removed while a node is attempting to state sync using it.
+
+- Steps:
+  - Enable state sync in `$HOME/.dcl/config/config.toml`:
+    ```
+    # RPC servers (comma-separated) for light client verification of the synced state machine and
+    # retrieval of state data for node bootstrapping. Also needs a trusted height and corresponding
+    # header hash obtained from a trusted source, and a period during which validators can be trusted.
+    #
+    # For Cosmos SDK-based chains, trust_period should usually be about 2/3 of the unbonding time (~2
+    # weeks) during which they can be financially punished (slashed) for misbehavior.
+
+    rpc_servers = "https://on.test-net.dcl.csa-iot.org:26657,https://on.test-net.dcl.csa-iot.org:26657"
+    trust_height = 429539
+    trust_hash = "F5D69C5BAFB69388980F6550CB68362CCF10491F8A98A96AB9008BBC186D48D4"
+    trust_period = "168h0m0s"
+    ```
+  - Provide `seeds` or `persistent_peers` in `$HOME/.dcl/config/config.toml`
```

Comment:
I think we need to provide very short and clear step-by-step commands how to run a new node via state sync for the Test Net assuming connection to CSA ONs.